src/views/index.js

```diff
@@ -121,6 +123,188 @@ const template = (...elements) => {
 
   return result;
 };
+
+const postInAside = msg => {
```

Comment:
so this is duplicating a lot of code from `post()`. My approach here would be to add a library with utilites for working with post objects to solve things like assembling the object of URL's for a post.

Reply:
This would be cool! I've been thinking that maybe the router (`src/index.js`) should be handing these URL builders, and I think koa-router even supports this.